### PR TITLE
Service Group Lifecycle Settings doc update

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

### :nut_and_bolt: Description: What code changed, and why?

[PR 4010](https://github.com/chef/automate/pull/4010) updated some documentation about the new browser-accessible Service Group Data Lifecycle settings. This PR finishes the documentation update. I also included some other copyediting for the modified pages.

### :chains: Related Resources

https://github.com/chef/automate/pull/4010/

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/automate-chef-io/content/docs/applications-dashboard.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ Applying the filter changes the Service Groups list to display matching results
 
 Disconnected services are defined services that have not reported back with a health check message for a user-defined period of time.
 
-Use the Chef Automate Service Groups API endpoint to manage the activity of periodic jobs, and to assess disconnection status and removal of services.
+Use the [Service Group Data Lifecycle settings](https://automate.chef.io/docs/data-lifecycle/#service-groups) to manage the activity of periodic jobs, and to assess disconnection status and removal of services.
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/applications-dashboard.md:98]: Question to y'all - With this recommendation of using the UI to manage settings in line 98, do we want to remove references of API management elsewhere in this page? See line 122 (Marking a Service as Disconnected) and line 126 (Periodic Removal)

File: components/automate-chef-io/content/docs/data-lifecycle.md [hunk 2]
```diff
@@ -30,16 +30,16 @@ Navigate to _Settings_ > _Data Lifecycle_ and adjust any settings you would like
...
 
-The event feed data lifecycle settings allow you to remove all event feed data and Chef Infra Server actions after a set amount of days.
+The Event Feed Data Lifecycle settings allow you to remove all event feed data and Chef Infra Server actions after a set amount of days.
 The default is to remove event feed data after 7 days, and Chef Infra Server actions after 30 days.
 
 ### Service Groups
 
-The service group data lifecycle settings allow you to label health check reports as disconnected and automatically remove them after a set amount of time.
+The Service Group Data Lifecycle settings allow you to label health check reports as disconnected and automatically remove them after a set amount of time.
 The default is to label health check reports as disconnected after 5 minutes, and remove disconnected services after 5 days.
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/data-lifecycle.md:43]: ALSO! I noticed that the screenshot in this page has label health check reports as disconnected after 5 minutes, and remove disconnected services after **7** days. Should we amend this line to say 7 days? The Periodic Removal section in applications-dashboard.md also references a 7 day default.

Comment (danielsdeleo) [reply components/automate-chef-io/content/docs/data-lifecycle.md:43]: Good catch! It is 7 days: https://github.com/chef/automate/blob/7da2e48913e6616e815278b1392f7df916e788e5/components/applications-service/pkg/server/periodic.go#L474

Comment (mjingle) [reply components/automate-chef-io/content/docs/data-lifecycle.md:43]: Added!

File: components/automate-chef-io/content/docs/applications-dashboard.md (hunk 1 above)
Comment (mjingle) [reply components/automate-chef-io/content/docs/applications-dashboard.md:98]: Content after line 122 is now removed! Thanks @danielsdeleo!

**Pull request merged**